docs/tutorial/dapr-ambient-and-knative.md

```diff
@@ -0,0 +1,201 @@
+- [Install `helm`](https://helm.sh/docs/intro/install/) 
+- [Install `docker`](https://docs.docker.com/engine/install/)
+- [Install the Knative Functions `func` CLI](https://knative.dev/docs/functions/install-func/)
+- [Install the `dapr` CLI](https://docs.dapr.io/getting-started/install-dapr-cli/)
+
+Create a local Kubernetes cluster with: 
+
+```
+cat <<EOF | kind create cluster --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraPortMappings:
+  - containerPort: 31080 # expose port 31380 of the node to port 80 on the host, later to be use by kourier or contour ingress
+    listenAddress: 127.0.0.1
+    hostPort: 80
+EOF
+```
+
+Let's now install Knative Serving into the cluster: 
+
+[Check this link for full instructions from the official docs](https://knative.dev/docs/install/yaml-install/serving/install-serving-with-yaml/#prerequisites)
+
+```
+kubectl apply -f https://github.com/knative/serving/releases/download/knative-v1.8.0/serving-crds.yaml
+kubectl apply -f https://github.com/knative/serving/releases/download/knative-v1.8.0/serving-core.yaml
+
+```
+
+Installing the networking stack to support advanced traffic management: 
+
+```
+kubectl apply -f https://github.com/knative/net-kourier/releases/download/knative-v1.8.0/kourier.yaml
+
+```
+
+```
+kubectl patch configmap/config-network \
+  --namespace knative-serving \
+  --type merge \
+  --patch '{"data":{"ingress-class":"kourier.ingress.networking.knative.dev"}}'
+
+```
+
+Configuring domain mappings: 
+
+```
+kubectl apply -f https://github.com/knative/serving/releases/download/knative-v1.8.0/serving-default-domain.yaml
+
+```
+
+**Only for Knative on KinD** 
+
+For Knative Magic DNS to work in KinD you need to patch the following ConfigMap:
+
+```
+kubectl patch configmap -n knative-serving config-domain -p "{\"data\": {\"127.0.0.1.sslip.io\": \"\"}}"
+```
+
+and if you installed the `kourier` networking layer you need to create an ingress:
+
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Service
+metadata:
+  name: kourier-ingress
+  namespace: kourier-system
+  labels:
+    networking.knative.dev/ingress-provider: kourier
+spec:
+  type: NodePort
+  selector:
+    app: 3scale-kourier-gateway
+  ports:
+    - name: http2
+      nodePort: 31080
+      port: 80
+      targetPort: 8080
+EOF
+```
+
+
+Let's create a Redis instance that our applications can use to store state or exchange messages: 
+
+```
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo update                            
+helm install redis bitnami/redis --set image.tag=6.2 --set architecture=standalone
+```
+
+Finally, let's install Dapr into the Cluster: 
+
+```
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo update
+helm upgrade --install dapr dapr/dapr \
+--version=1.10.4 \
+--namespace dapr-system \
+--create-namespace \
+--wait
+```
+
+Let's now deploy and configure some Dapr apps! 
+
```

Comment:
@mcruzdev here add the instructions to deploy `dapr-ambient` using the helm chart and set the remoteURL to the subscriber service in kubernetes: 

https://github.com/salaboy/dapr-ambient-examles